src/Slicing.jl

```diff
@@ -146,10 +146,26 @@ Base.@kwdef struct FlopsScorer <: Scorer
 end
 
 function (cb::FlopsScorer)(path, index)
-    slice = selectdim(path, index, 1)
+    total_reductions = mapreduce(
+        sexpr -> (
+            flops(sexpr) - filtered_flops(sexpr, index),
+            length(sexpr) - filtered_length(sexpr, index)
+        ),
+        (a, b) -> (a[1] + b[1], a[2] + b[2]),
```

Comment:
i think you can just put `.+` here
```suggestion
        .+,
```

`+` doesn't work with `Tuple`s but `.+` does (and returns a `Tuple`)

also, it seems like `@inbound` would make no effect here

```julia
julia> f_nobounds(a,b) = .+(a,b)

julia> f_inbounds(a,b) = @inbounds .+(a,b)

julia> using Chairmarks

julia> @b (BigInt(1),BigInt(2)), (BigInt(3),BigInt(4)) splat(f_nobounds)
41.003 ns (4 allocs: 96 bytes)

julia> @b (BigInt(1),BigInt(2)), (BigInt(3),BigInt(4)) splat(f_inbounds)
40.644 ns (4 allocs: 96 bytes)
```